README.md

```diff
@@ -100,3 +100,30 @@ Please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file for more information
 
 
 [helm]: https://helm.sh/
+
+# Deploy RHTAP
+Follow the below steps to deploy RHTAP on Openshift cluster.
+
+1.Clone the repositry.Run the command 'make` from the rhtap-cli directory.
```

Comment:
Could you add spaces after the `.` (dots), please? All the list is lacking space after dot/period, please consider.